src/sidebar/websocket.js

```diff
@@ -1,5 +1,5 @@
 import retry from 'retry';
-import EventEmitter from 'tiny-emitter';
+import { TinyEmitter } from 'tiny-emitter';
```

Comment:
`TinyEmitter` is exactly the same as `EventEmitter`, but the built-in type information in the `tiny-emitter` package only includes the `TinyEmitter` class.

Reply:
This begs the question of type-checking integration with third-party libraries...what happens if a dependency we use doesn't provide type info?

Reply:
Broadly, there are 4 options:

1. Import a package of type info that someone else wrote. There is a project called [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) that publishes many of these. I plan to do this for "mocha" and "chai".
2. Write our own type info for the library
3. Tell TS to ignore the import
4. Use a different library that does have typo info